### ISSUE TITLE
A11y_VisualStudioCodeClient_Sidebar Bar Options_ Screen reader: Voiceover is announcing the setting button as group button.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180114]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG4.1.2; #Voiceover; #Screenreader; #Name Role Value.

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor
2. .Using "Command+Shift+X" key navigate to the extension side view bar option.
3. Tab till "Setting" button near any extension.
4. Verify that Voiceover is announcing the setting button as group button.

### Actual Result:
Voiceover is announcing the setting button as group button.

### Expected Result:
Voiceover should announce the button as "Setting button"

### ![197950986-40c9ef7e-39b3-466c-80c5-108bbf23d17e](https://user-images.githubusercontent.com/104351566/232458275-ecbd569f-e587-473c-a569-dea6d316e6dd.gif)

**Comment by BhanuPrakashGit** [comment]
Reference bug: https://github.com/Microsoft/vscode/issues/164698#issue-1423847002

**Comment by meganrogge** [comment]
@kjawale99 VS Code can not control this. I suggest you file a new issue for VoiceOver https://bugs.webkit.org/

**Comment by BhanuPrakashGit** [comment]
GithubBug:#Regressed:04-17-23
Logged IRT bug against VoiceOver.
